WordPress/Classes/ViewRelated/NUX/LoginEmailViewController.swift

```diff
@@ -198,7 +198,7 @@ class LoginEmailViewController: LoginViewController, SigninKeyboardResponder {
         loginFields.safariStoredUsernameHash = username.hash
         loginFields.safariStoredPasswordHash = password.hash
 
-        loginWithUsernamePassword(immediately: false)
+        loginWithUsernamePassword(immediately: true)
```

Comment:
👍, weird that we weren't doing this for safari stored passwords previously when we were doing it for 1password.